lucida/commandcenter/controllers/Config.py

```diff
@@ -12,29 +13,45 @@
 
 # Pre-configured services.
 # The ThriftClient assumes that the following services are running.
-# Host IP addresses are resolved dynamically: 
+# Host IP addresses are resolved dynamically:
 # either set by Kubernetes or localhost.
-SERVICES = { 
-	'IMM' : Service('IMM', 8082, 'image', 'image'), 
-	'QA' : Service('QA', 8083, 'text', 'text'),
-	'CA' : Service('CA', 8084, 'text', None),
-	'IMC' : Service('IMC', 8085, 'image', None),
-	'FACE' : Service('FACE', 8086, 'image', None),
-	'DIG' : Service('DIG', 8087, 'image', None),
-	'ENSEMBLE' : Service('ENSEMBLE', 9090, 'text', None) 
-	}
+SERVICES = {
+    'IMM' : Service('IMM', 8082, 'image', 'image'),
+    'QA' : Service('QA', 8083, 'text', 'text'),
+    'CA' : Service('CA', 8084, 'text', None),
+    'IMC' : Service('IMC', 8085, 'image', None),
+    'FACE' : Service('FACE', 8086, 'image', None),
+    'DIG' : Service('DIG', 8087, 'image', None),
+    'ENSEMBLE' : Service('ENSEMBLE', 9090, 'text', None),
+    'WE' : Service('WE', 8088, 'text', None),
+    'DCM_WE' : WorkerService('DCM', Decision().class_WE_DCM),
+    'DCM_IMM' : WorkerService('DCM', Decision().class_IMM_DCM)
+    }
 
 # Map from input type to query classes and services needed by each class.
-CLASSIFIER_DESCRIPTIONS = { 
-	'text' : { 'class_QA' :  Graph([Node('QA')]) ,
-			   'class_CA' : Graph([Node('CA')]) },
-	'image' : { 'class_IMM' : Graph([Node('IMM')]),
-				'class_IMC' : Graph([Node('IMC')]),
-				'class_FACE' : Graph([Node('FACE')]),
-				'class_DIG' : Graph([Node('DIG')]) },
-	'text_image' : { 'class_QA': Graph([Node('QA')]),
-					 'class_IMM' : Graph([Node('IMM')]), 
-					 'class_IMM_QA' : Graph([Node('IMM', [1]), Node('QA')]),
-					 'class_IMC' : Graph([Node('IMC')]),
-					 'class_FACE' : Graph([Node('FACE')]),
-					 'class_DIG' : Graph([Node('DIG')]) } }
+CLASSIFIER_DESCRIPTIONS = {
+    'text' : { 'class_QA' :  Graph([Node('QA')]),
+               'class_CA' : Graph([Node('CA')]),
+               'class_WE' : Graph([Node('WE')]),
+               'class_WE_DCM' : Graph([Node('WE', [1]), Node('DCM_WE', [0])]) },
+    'image' : { 'class_IMM' : Graph([Node('IMM')]),
+                'class_IMC' : Graph([Node('IMC')]),
+                'class_FACE' : Graph([Node('FACE')]),
+                'class_DIG' : Graph([Node('DIG')]) },
+    'text_image' : { 'class_QA': Graph([Node('QA')]),
+                     'class_IMM' : Graph([Node('IMM')]),
+                     'class_IMM_QA' : Graph([Node('IMM', [1]), Node('QA')]),
+                     'class_IMC' : Graph([Node('IMC')]),
+                     'class_FACE' : Graph([Node('FACE')]),
+                     'class_DIG' : Graph([Node('DIG')]),
+                     'class_IMM_DCM_QA_WE' : Graph([Node('IMM', [1]), Node('DCM_IMM', [0,2]), Node('QA', [3]), Node('WE')]) }
+    }
+
+# TODO: Should I have this in its own Config file?
```

Comment:
Yes, I need this for multi-turn paths. Do you think it makes sense to keep `SESSION` contained in this Config file or create a separate file since developers using Lucida don't necessarily need to know about this global?